From 56a7ddde373fcd96cb02b00774cf66e347b318c3 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Tue, 4 Sep 2018 09:50:55 +0900
Subject: [PATCH 519/909] arm64: dts: salvator-common: Set max dot clock for
 vga

commit ab99897bb0960d6988ae72ce156379e7187fce2a from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Add option to specify the maximum dot clock that can be
output on VGA connector in salvator board.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/salvator-common.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/salvator-common.dtsi b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
index a538fe295d9c..5f586d77f7dd 100644
--- a/arch/arm64/boot/dts/renesas/salvator-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
@@ -205,6 +205,7 @@
 	vga {
 		compatible = "vga-connector";
 		no-use-ddc;
+		max-pixelclock = <100000>;
 
 		port {
 			vga_in: endpoint {
-- 
2.17.1

